src/tools/pontoon/teams_projects.md

```diff
@@ -73,36 +78,53 @@ It also includes an indication of the overall status of completion and statistic
 * **Info**: information about this project.
+* **Notifications**: contains functionality to notify those who have contributed to this project for defined locales.
 
 ### Requesting a language
 
-It’s possible to request an additional language for a project from its *Project page*.
+It’s possible to request an additional language for some projects from its *Project page*.
 
 ![Request a project](../../assets/images/pontoon/teams_projects/request_locale.png)
 
-Click on *REQUEST NEW LANGUAGE* (highlighted in red), select the language you want to add and then click *REQUEST NEW LANGUAGE*.
+Click on `REQUEST NEW LANGUAGE` (highlighted in red), select the language to add and then click `REQUEST NEW LANGUAGE`.
 
 An email will be sent to Pontoon’s administrators, and the localization driver in charge of the project will act on the request. The same restrictions described in the [Requesting a project](#requesting-a-project) section apply here.
 
 ## Localization page
 
-You can access a project’s *Localization page* in two ways:
+Access a project’s *Localization page* in two ways:
 * By selecting it from the [Team page](#team-page).
 * By selecting it from the [Project page](#project-page).
 
 The page header contains useful information about the current project (same as the [Project page](#project-page)).
 
 ![Header of localization page](../../assets/images/pontoon/teams_projects/localization_page_header.png)
 
-There are 4 tabs available (highlighted in red):
+There are 6 tabs available (highlighted in red):
 * **Resources**: files available in the project.
 * **Tags**: a list of tags defined for this project. This tab will be visible only in projects where tags are enabled. For more information about tags, see [this page](search_filters.md#tags).
 * **Contributors**: a list of active contributors with their statistics, filterable by time. Unlike the tab available in the *Team page*, this only includes contributions to localization of the project for the current locale.
+* **Insights**: contains data and trends presented in a graphical format about review activity and translation activity.
 * **Project info**: information about the project.
 * **Team info**: same content as the *Info* tab in the *Team page*.
+
+In addition to the *Localization page* for each locale/project pair, there is also a special Localization page that allows access to all resources for all projects enabled for a specific locale. This can be accessed from the `/{LOCALE}/all-projects/all-resources` URL (e.g. [pontoon.mozilla.org/it/all-projects/all-resources](https://pontoon.mozilla.org/it/all-projects/all-resources) for Italian).
+
+## Insights graphs
+
+The *Insights tab*, accessible from either the *Project*, *Team*, or *Localization page*, displays data and trends on contributor and localization activity in a graphical format. Depending on the page, the following insights may be shown:
```

Comment:
Instead of repeating `(Only appears on Team page)`, I would split those 3 graphs in a separate list.

Intro for the first list `The following insights are displayed in all different pages:`, for the new list `These insights are displayed only in the Team page:`.